### ISSUE TITLE
Certain projectiles not giving logs

### ISSUE THREAD

**Issue by Vakothu** [UristMcStation/UristMcStation#495]
I can't figure out the exact reason, but the Pulse Cannon projectile from SCOM (/obj/item/projectile/beam/pulse/heavy/h2) as well as its parent projectile (/obj/item/projectile/beam/pulse/heavy) which is fired from the eZ-13 mk2 heavy pulse rifle for mechs do not give admin combat logs, nor leave a log in the victim's variables.
I'd suggest checking around code/game/mecha/equipment/weapons/weapons.dm line 111.  Looks like old code, but I'm not 100% sure.

**Comment by Vakothu** [comment]
Forgot to update this: Scr checked it out, and /obj/item/projectile/beam/pulse/heavy totally bypasses normal gun-code.

**Comment by scrdest** [comment]
Yeah, it's super-hacky: it hooks into movement procs themselves (Bump() IIRC) and directly qdels targets.
